app/views/handlers/individual_response.py

```diff
@@ -0,0 +1,335 @@
+        list_model = self._questionnaire_store.list_store[list_name]
+
+        if self._list_item_id:
+            if self._list_item_id not in list_model:
+                abort(404)
+
+            if self._list_item_id == list_model.primary_person:
+                raise InvalidLocationException(
+                    "individual response not accessible for primary person"
+                )
+
+    @cached_property
+    def rendered_block(self) -> Mapping:
+        return self._render_block()
+
+    @cached_property
+    def placeholder_renderer(self):
+        return PlaceholderRenderer(
+            language=self._language,
+            schema=self._schema,
+            answer_store=self._questionnaire_store.answer_store,
+            metadata=self._questionnaire_store.metadata,
+            location=None,
+            list_store=self._questionnaire_store.list_store,
+        )
+
+    @cached_property
+    def router(self):
+        return Router(
+            schema=self._schema,
+            answer_store=self._questionnaire_store.answer_store,
+            list_store=self._questionnaire_store.list_store,
+            progress_store=self._questionnaire_store.progress_store,
+            metadata=self._questionnaire_store.metadata,
+        )
+
+    @cached_property
+    def can_access_hub(self):
+        return self.router.can_access_hub()
+
+    @cached_property
+    def individual_section_id(self):
+        return self._schema.json.get("individual_response", {}).get(
+            "individual_section_id"
+        )
+
+    @cached_property
+    def form(self):
+        return generate_form(
+            schema=self._schema,
+            question_schema=self._block_definition["question"],
+            answer_store=None,
+            metadata=self._questionnaire_store.metadata,
+            form_data=self._form_data,
+        )
+
+    def get_context(self):
+        return build_question_context(self.rendered_block, self.form)
+
+    def handle_get(self):
+        list_name = self._schema.json.get("individual_response", {}).get("for_list")
+
+        previous_location_url = url_for(
+            "questionnaire.block",
+            list_name=list_name,
+            list_item_id=self._list_item_id,
+            block_id=self.individual_section_id,
+        )
+
+        return render_template(
+            template="individual_response/interstitial",
+            language=self._language,
+            previous_location_url=previous_location_url,
+        )
+
+    def _render_block(self):
+        transformed_block = transform_variants(
+            self._block_definition,
+            self._schema,
+            self._questionnaire_store.metadata,
+            self._questionnaire_store.answer_store,
+            self._questionnaire_store.list_store,
+            None,
+        )
+
+        return self.placeholder_renderer.render(transformed_block, self._list_item_id)
+
+
+class IndividualResponseHowHandler(IndividualResponseHandler):
```

Comment:
The submit button text during the IR journey in the prototype is `Continue` vs our `Save and continue`.